web/pandas/about/team.md

```diff
@@ -8,31 +8,27 @@ If you want to support pandas development, you can find information in the [dona
 
 ## Maintainers
 
-<div class="row maintainers">
-    {% for row in maintainers.people | batch(6, "") %}
-        <div class="card-group maintainers">
-            {% for person in row %}
-                {% if person %}
-                    <div class="card">
-                        <img class="card-img-top" alt="" src="{{ person.avatar_url }}"/>
-                        <div class="card-body">
-                            <h6 class="card-title">
-                                {% if person.blog %}
-                                    <a href="{{ person.blog }}">
-                                        {{ person.name or person.login }}
-                                    </a>
-                                {% else %}
-                                    {{ person.name or person.login }}
-                                {% endif %}
-                            </h6>
-                            <p class="card-text small"><a href="{{ person.html_url }}">{{ person.login }}</a></p>
-                        </div>
-                    </div>
-                {% else %}
-                    <div class="card border-0"></div>
-                {% endif %}
-            {% endfor %}
-        </div>
+<div class="card-group maintainers">
+    {% for person in maintainers.people %}
+        {% if person %}
```

Comment:
If I'm not wrong, this was the `if` used to discriminate the maintainers from the empty loop iterations generated by Jinja's `batch`, so not needed anymore. All the maintainers are people now ;)

Reply:
Good catch, thanks. My bad. I'll make the change on the inside as well.